### ISSUE TITLE
Wrong hovering when afternoonHoursInOuterCircle is set to true

### ISSUE THREAD

**Issue by sabat24** [loebi-ch/jquery-clock-timepicker#16]
As stated in title. If you turn on that option hovering works oposite. If you move cursor over 13, 1 will be highlighted. And if you move mouse over 1, 13 will be highlighted.

I added following code in processTimeSelection method

```
if (settings.afternoonHoursInOuterCircle) {
	h = h + (h >= 12 ? -12 : 12);
}
```

just before

`if (h > -1) {`

**Comment by loebi-ch** [comment]
Bug solved with version 2.3.0

Thank you very much for taking the time to provide me with the solution :-)

**Comment by sabat24** [comment]
It was a pleasure to work with your plugin :) You did a great job and code was very clean.
